src/components/profile/profile.pw.tsx

```diff
@@ -59,58 +61,178 @@ test.describe("Prop checks for Profile component", () => {
     });
   });
 
-  ([
```

Comment:
**question**: it looks like these tests, checking the appropriate `width` and `height` styles exist depending on the `size` prop, have been removed. Is there a good reason for that? Particularly as I see you've added tests for things like `line-height`, which I'm not saying not to do but seem a less important CSS property to test than width or height.

Reply:
Good question, I felt these were redundant though as we're testing the `Portait` component here, when I feel there's already enough tests in that file for sizes, we actually cover every type of prop in `portrait.pw.tsx` whereas this previous test was only testing the `initials` prop

Reply:
Essentially, I felt repeating the test here was redundant

Reply:
I see, I'd missed that these were effectively testing `portrait`. Happy to leave this for the next reviewer to comment on, on the one hand I agree with you, but on the other the fact that `profile` sometimes renders `portrait` could be argued to be just an implementation detail, and that if we ever refactored that we'd still want to be sure `profile` was rendering at the correct size. (Mind you Chromatic should catch any changes there.) So there's an argument both ways - personally given the uncertainty I'd always rather have too many tests than too few, but as I said I'm happy to wait for another opinion and don't object to dropping this.

Reply:
As long as we have some way of catching if this were to regress (ie passing size to Profile doesn't set Portrait) I don't mind where they are

Reply:
I'll look into this as flex-shrink: 0 is definitely needed now we've moved to a new approach for `initials`, moving away from rendering an image and instead rendering a div with centred text etc. I'll make sure its applied in the correct container, and on @edleeks87's point only apply it when `Portrait` is a child of `Profile`. Apologies for mixing the names up but as you can imagine working on Profile/Portrait for a while soon its quite easy to mix them up haha